manim/cli/render/commands.py

```diff
@@ -0,0 +1,386 @@
+    multiple=True,
+    help="Specify the filename(s) of the rendered scene(s).",
+)
+@optgroup.option(
+    "--write_to_movie",
+    is_flag = True,
+    help="Write to a file.",
+)
+@optgroup.option(
+    "--media_dir", type=click.Path(), help="Path to store rendered videos and latex."
+)
+@optgroup.option("--log_dir", type=click.Path(), help="Path to store render logs.")
+@optgroup.option(
+    "--log_to_file",
+    default=True,
+    show_default=True,
+    is_flag=True,
+    help="Log terminal output to file",
+)
+@optgroup.group("Render Options")
+@optgroup.option(
+    "-n",
+    "--from_animation_number",
+    callback=validate_scene_range,
+    help="Start rendering from n_0 until n_1. If n_1 is left unspecified, renders all scenes after n_0.",
+)
+@optgroup.option(
+    "-a",
+    "--write_all",
+    is_flag=True,
+    help="Render all scenes in the input file.",
+)
+@optgroup.option(
+    "-f",
+    "--format",
+    default="mp4",
+    type=click.Choice(
+        [
+            "png",
+            "gif",
+            "mp4",
+        ],
+        case_sensitive=False,
+    ),
+)
+@optgroup.option("-s", "--save_last_frame", is_flag=True)
+@optgroup.option(
+    "-q",
+    "--quality",
+    default="h",
+    type=click.Choice(
+        [
+            "l",
+            "m",
+            "h",
+            "p",
+            "k",
+        ],
+        case_sensitive=False,
+    ),
+    help=
+        """
+        Render quality at the follow resolution framerates, respectively:
+        854x480 30FPS, 
+        1280x720 30FPS,
+        1920x1080 60FPS,
+        2560x1440 60FPS,
+        3840x2160 60FPS
+        """
+    ,
+)
+@optgroup.option(
+    "-r",
+    "--resolution",
+    callback=validate_resolution,
+    help="Resolution in (W,H) for when 16:9 aspect ratio isn't possible.",
+)
+@optgroup.option(
+    "--fps",
+    "--frame_rate",
+    "frame_rate",
+    type=float,
+    help="Render at this frame rate.",
+)
+@optgroup.option(
+    "--opengl_renderer",
+    is_flag = True,
+    help="Render scenes using OpenGL."
+)
+@optgroup.option(
+    "--webgl_renderer",
+    default=None,
+    type=click.Path(),
+    help="Render scenes using the WebGL frontend. Requires a path to the WebGL frontend.",
+)
+@optgroup.option(
+    "-t", "--transparent", is_flag=True, help="Render scenes with alpha channel."
+)
+@optgroup.option(
+    "-c",
+    "--background_color",
+    show_default=True,
+    default="#000000",
+    help="Render scenes with background color.",
+)
+@optgroup.group("Ease of access options")
+@optgroup.option(
+    "--progress_bar",
+    default="display",
+    show_default=True,
+    type=click.Choice(
+        [
+            "display",
+            "leave",
+            "none",
+        ],
+        case_sensitive=False,
+    ),
+    help="Display progress bars and/or keep them displayed.",
+)
+@optgroup.option(
+    "-p",
+    "--preview",
+    is_flag=True,
+    help="Preview the rendered file(s) in default player.",
+)
```

Comment:
What would you suggest changing the help message to?

Reply:
If OpenGL renderer is used `-p` would pop up a window with animation renderer. If Cairo renderer is used, it would just play in media player after renderer.